[Flaky CI]: test_metrics.py::test_global_metrics_with_processing

### Flakiness Type

Assertion failure

### Name of Test

tests/integration/test_metrics.py::test_global_metrics_with_processing

### Link to Test Run

https://github.com/getsentry/relay/actions/runs/7885499882/job/21516771583

### Details

Looks like timestamp mismatch. 


```
=================================== FAILURES ===================================
_____________________ test_global_metrics_with_processing ______________________
[gw0] linux -- Python 3.10.13 /home/runner/work/relay/relay/.venv/bin/python3
Traceback (most recent call last):
  File "/home/runner/work/relay/relay/tests/integration/test_metrics.py", line 394, in test_global_metrics_with_processing
    assert metrics["c:transactions/foo@none"] == {
AssertionError: assert {'name': 'c:t...ays': 90, ...} == {'name': 'c:t...ays': 90, ...}
  Omitting 7 identical items, use -vv to show
  Differing items:
  {'timestamp': 1707822482} != {'timestamp': 1707822481}
  Full diff:
    {
     'name': 'c:transactions/foo@none',
     'org_id': 1,
     'project_id': 42,
     'retention_days': 90,
     'tags': {},
  -  'timestamp': 1707822481,
  ?                        ^
  +  'timestamp': 1707822482,
  ?                        ^
     'type': 'c',
     'value': 42.0,
    }
```

I have a failure of `FAILED tests/integration/test_outcome.py::test_global_rate_limit_by_namespace` [here](https://github.com/getsentry/relay/actions/runs/7887800311/job/21523869205).

And we have #3030 as well.

--- a/tests/integration/test_metrics.py
+++ b/tests/integration/test_metrics.py
@@ -487,7 +487,9 @@ def test_global_metrics_with_processing(
     project_config["config"]["features"] = ["organizations:custom-metrics"]
 
     timestamp = int(datetime.now(tz=timezone.utc).timestamp())
-    metrics_payload = f"transactions/foo:42|c\nbar@second:17|c|T{timestamp}"
+    metrics_payload = (
+        f"transactions/foo:42|c|T{timestamp}\nbar@second:17|c|T{timestamp}"
+    )
     relay.send_metrics(project_id, metrics_payload)
 
     metrics = metrics_by_name(metrics_consumer, 2)

